package.json

```diff
@@ -24,7 +24,8 @@
   },
   "dependencies": {
     "react": "^16.3.2",
-    "react-native": "^0.55.4"
+    "react-native": "^0.55.4",
+    "react-native-share": "^1.0.27"
```

Comment:
@tbergq Wrong file but not sure where you need it exactly. Shared or MMB?

Reply:
Right, for now it will be in MMB

Reply:
To be specific, this should be added to `app/MMB/package.json`